test/compflags/bradc/help/userhelp.good

```diff
@@ -22,6 +22,7 @@ Optimization Control Options:
       --[no-]dead-code-elimination    Enable [disable] dead code elimination
       --fast                          Use fast default settings
       --[no-]fast-followers           Enable [disable] fast followers
+      --[no-]force-initializers       Enable [disable] default initializers
```

Comment:
I guesssssssss?  I don't want to put the tests into the deprecated directory yet, I think.  I'm not yet sure on the time frame for removing the default constructor code and so might keep it around longer, perhaps as long as the deprecation warning for explicit constructors.  If that's the case, it might be useful for it to be a flag a user could use; if they find a problem with default initializers, we might want to have them see if they still have that problem with default constructors and so might tell them to use that flag.